[CS2103T-F14-1] DukeAcademy

zhangj1an opened the pull request:
@alxkohh @ning-sy210 @limhawjia

ChrisKheng commented on docs/DeveloperGuide.adoc:
```diff
@@ -295,71 +316,174 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 [discrete]
-=== Use case: Delete person
-
+=== Use case: UC01 Set questions
```
This use case is well-written!

dcshzj commented on docs/DeveloperGuide.adoc:
```diff
@@ -295,71 +316,174 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+** 2c1. Duke Academy requests to save the draft.
+** 2c2. User confirms or denies.
+** 2c3. Duke Academy follows user's preference to save or discard the draft.
+** 2d4. Duke Academy exists.
```
Is this word correct? Should it be "exits" instead of "exists?

dcshzj commented on docs/DeveloperGuide.adoc:
```diff
@@ -295,71 +316,174 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+* *a. At any time, user chooses to attempt an question imported from external resources.
++
+[none]
+** *a1. load the questions from file.
```
Is this format correct? Should it start with a capital letter "L" instead?

dcshzj commented on docs/DeveloperGuide.adoc:
```diff
@@ -295,71 +316,174 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
++
+*a3. User search for the problem imported.
++
+*a4. Duke Academy displays the question.
```
Is this format correct? Should it end/resume the use case instead?

dcshzj commented on docs/DeveloperGuide.adoc:
```diff
@@ -295,71 +316,174 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+[none]
+* 4a. The input format is incorrect.
++
+Duke Academy reports wrong format error. Duke Academy resumes at step 3.
```
Is this format correct? Should it be separated into 2 lines instead?

ChrisKheng commented on docs/DeveloperGuide.adoc:
```diff
@@ -295,71 +316,174 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+*Extensions*
 [none]
-* 2a. The list is empty.
+* 1a. User inputs wrong keywords.
```
I think instead of 'user inputs wrong keywords', it should be 'user's inputted keyword cannot be found' because technically I think the keyword cannot be wrong since at most is the keyword cannot be found, so maybe can show a "no question found" message if this occurs.

mrchensheng21 commented on docs/DeveloperGuide.adoc:
```diff
@@ -295,71 +316,174 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+
+|`* *` |coder|identify the concepts required to solve a problem before attempting them |move on to another quickly
+
+|`* *` |a coding student|attempt the same problem in different coding languages|test my proficiency at those languages
+
+|`* *` |coding amateur|look at hints/tutorials for the problem|learn something new while attempting a coding challenge
```
Just a minor change. Coding student and coding amateur is not consistent, should put it as "student coder" and "amateur coder"

ChrisKheng commented on docs/DeveloperGuide.adoc:
```diff
@@ -295,71 +316,174 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
++
+Use case resumes from step 4.
+
+* 2c. User requests to quit the program.
```
This Use case is ok!

ChrisKheng reviewed (commented):
In general, the team uses a lot of different types of diagrams to accompany the description which aids the description. However, there are some flaws in some of the diagrams: 
1. Storage component ->Maybe the question bank storage interface can be put on top of JsonQuestionBankStorage to make the interface relationship clearer.

2. Sequence diagram under 3.3.2 
-> There should be a return arrow after clearEnvironment because for the subsequent method class there are return arrows, except this one. So, the clearEnvironment should have a return arrow to ensure consistency.

-> The labels at the bottom of the sequence diagram to indicate the names of the classes can be removed since there are already labels on tops of the sequence diagram serving the same purpose.

Nevertheless, the team did a good job in including different types of diagrams to accompany the description!

cheongsiuhong commented on docs/DeveloperGuide.adoc:
```diff
@@ -227,20 +250,159 @@ image::CommitActivityDiagram.png[]
+
+The Sequence Diagram given below is an example of a scenario in which the submit command is executed when the user enters "submit" into the `CommandBox` terminal.
+
+image::submitSequenceDiagram.png[]
```
Activation bar for command parser should end once it returns.
CommandParser and Command should be SubmitCommandParser and Submit Command.
Weird box on the right, bug?
Dotted line from command should end at the cross but I think it's a plantUML bug.

cheongsiuhong commented on docs/DeveloperGuide.adoc:
```diff
@@ -227,20 +250,159 @@ image::CommitActivityDiagram.png[]
+
+6. `TestExecutor` packs all the data collected from the execution of the test cases into a single `TestResult` instance.
+
+image::TestExecutorSequenceDiagram.png[align:center]
```
There should be colons before all the class names.
Return arrows should be dotted arrows.
Activation bar should stop after return.
Loop [test cases] can be more clear; like loop [while there are more test cases].
Why are there class names are the bottom of the diagram?

cheongsiuhong reviewed (commented):
Apart from small quirks in the diagrams, nice work.

limhawjia committed:
Remove checkbox from question card

limhawjia committed:
Change tabs to unclosable, remove redundant buttons

Commit:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-F14-1/main

limhawjia committed:
Make browse and find commands navigate to questions, make homepage read list of all questions

limhawjia committed:
Made question cards not selectable

limhawjia committed:
Merge pull request #112 from AY1920S1-CS2103T-F14-1/hj-tabs

Hj tabs

alxkohh committed:
Edit json to display formatted and guided boilerplate code for user

limhawjia committed:
Merge pull request #114 from AY1920S1-CS2103T-F14-1/ak-json

Ak json

alxkohh committed:
Fix profile picture and add portfolio link

alxkohh committed:
Add portfolio adoc for all members

alxkohh committed:
Update project overview and name in sample portfolio template

alxkohh committed:
Merge branch 'master' into ak-about-us

limhawjia committed:
Update ug (#115)

limhawjia committed:
Merge branch 'master' into ak-about-us

limhawjia committed:
Merge pull request #116 from AY1920S1-CS2103T-F14-1/ak-about-us

Ak about us

Commit:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-F14-1/main

alxkohh committed:
Allow program evaluation panel to display errors

alxkohh committed:
Add javadoc comments for new methods

limhawjia committed:
Fixed newline issue (#120)

alxkohh committed:
Merge branch 'master' into ak-display-error

limhawjia committed:
Merge pull request #119 from AY1920S1-CS2103T-F14-1/ak-display-error

Allow program evaluation panel to display errors

alxkohh committed:
Add delete bookmark command

alxkohh committed:
Fix checkstyle

limhawjia committed:
Merge pull request #123 from AY1920S1-CS2103T-F14-1/ak-delete-bookmark

Ak delete bookmark command

Commit:
Updated user guide

Commit:
Added new screenshots of User Interface to facilitate the UG

alxkohh committed:
Merge branch 'master' into sy-update-ug

alxkohh committed:
Update readme

ning-sy210 committed:
Updated UG with current state of application (#124)

* Included .class files in gitignore

* Updated user guide

* Added new screenshots of User Interface to facilitate the UG

alxkohh committed:
changed image on readme

alxkohh committed:
remove duplicate image

alxkohh committed:
fix alignment

alxkohh committed:
Merge branch 'master' into ak-readme

alxkohh committed:
Change Ui.png

alxkohh committed:
Merge branch 'ak-readme' of https://github.com/AY1920S1-CS2103T-F14-1/main into ak-readme

ning-sy210 committed:
Merge pull request #125 from AY1920S1-CS2103T-F14-1/ak-readme

Ak readme

Commit:
Changed UG with more updates

Commit:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-F14-1/main

Commit:
Updated UG with updated screenshots

Commit:
More updates to UG

Commit:
Merge branch 'sy-update-ug' of https://github.com/AY1920S1-CS2103T-F14-1/main into sy-update-ug

Commit:
Merge branch 'master' into sy-update-ug

ning-sy210 committed:
Sy update ug (#128)

* Included .class files in gitignore

* Updated user guide

* Added new screenshots of User Interface to facilitate the UG

* Changed UG with more updates

* Updated UG with updated screenshots

* More updates to UG

Commit:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-F14-1/main

Commit:
Removed "..." at the end of bookmark and deletebookmark function

Commit:
Merge branch 'sy-update-ug' of https://github.com/AY1920S1-CS2103T-F14-1/main into sy-update-ug

alxkohh committed:
Add unit tests for BookmarkCommand and BookmarkCommandFactory

ning-sy210 committed:
Merge branch 'master' into sy-update-ug

ning-sy210 committed:
Merge pull request #129 from AY1920S1-CS2103T-F14-1/sy-update-ug

Sy update ug

alxkohh committed:
Edit json to fix newline at EOF issue (#126)

limhawjia committed:
Hj async program execution (#130)

* Set testcase evaluation to timeout

* Fixed unit tests

* Add fix for platform-dependant new line

ning-sy210 committed:
Fix unfound json file in jar

ning-sy210 committed:
Adjusted proportions of ProblemStatementDisplay and ProgramEvaluationDisplay (#131)

Commit:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-F14-1/main

ning-sy210 committed:
Merge branch 'master' into hj-fix-jar-issues

limhawjia committed:
Merge pull request #132 from AY1920S1-CS2103T-F14-1/hj-fix-jar-issues

Fix issues with jar release

alxkohh committed:
Add index display for listviews on dashboard

zhangj1an committed:
Merge pull request #133 from AY1920S1-CS2103T-F14-1/ak-dashboard

Add index display beside questions

limhawjia committed:
Hj fix bugs (#135)

* Fixed bug with empty program

* Tried fix for user directory with space

* Changed storage to store at containing folder

* Set logs to output to a folder

alxkohh committed:
Fix some issues from mock pe

alxkohh committed:
Edit json for better ui presentation

alxkohh committed:
Fix newline issue

alxkohh committed:
Fix log file

alxkohh committed:
Remove local data

alxkohh committed:
Restart log

alxkohh committed:
Add help page and remove old help window

alxkohh committed:
Add Help activity into Activity Enum

alxkohh committed:
Refactor Command Result

alxkohh committed:
Remove unnecessary fxml files

alxkohh committed:
Rename fxml

alxkohh committed:
Refine javadoc for Home Command and Help Command

alxkohh committed:
Refine ui

alxkohh committed:
Fix all failed tests

alxkohh committed:
Fix typo

limhawjia committed:
Fix compile error not showing full message

alxkohh committed:
Added directories to gitignore

limhawjia committed:
Merge pull request #175 from AY1920S1-CS2103T-F14-1/ak-help

Ak help tab and help command

limhawjia committed:
Add  feedback for toggling tabs with click

alxkohh committed:
Merge branch 'master' into ak-pe-fix

limhawjia committed:
Add question id

zhangj1an committed:
Merge pull request #173 from AY1920S1-CS2103T-F14-1/ak-pe-fix

Ak pe fix for bookmark command

limhawjia committed:
Refactor tests and commands to fix new question class

limhawjia committed:
Merge remote-tracking branch 'origin/master' into hj-fix-bugs-2

# Conflicts:
#	DukeAcademy/data/QuestionBank.json
#	src/main/java/com/dukeacademy/logic/commands/bookmark/BookmarkCommand.java
#	src/main/java/com/dukeacademy/logic/commands/bookmark/DeleteBookmarkCommand.java
#	src/main/java/com/dukeacademy/logic/commands/tab/TabCommand.java
#	src/main/java/com/dukeacademy/logic/commands/view/ViewCommand.java
#	src/main/java/com/dukeacademy/ui/MainWindow.java

limhawjia committed:
Merge pull request #176 from AY1920S1-CS2103T-F14-1/hj-fix-bugs-2

Hj fix bugs 2

zhangj1an committed:
temp change on home command

zhangj1an committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-F14-1/main

zhangj1an committed:
updated ppp

zhangj1an committed:
added reposense code link

zhangj1an committed:
Merge pull request #178 from AY1920S1-CS2103T-F14-1/zj-unit-tests

Zj updated ppp

limhawjia committed:
Hj fix bugs 3 (#180)

* Fix wrong update of success result

* Remove redundant filter clearing

* Fix bug with main class being commented out

* Added logs folder to gitignore

Commit:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-F14-1/main

limhawjia committed:
Hj notes (#182)

* Add logs folder to gitignore

* Add skeleton for notes page

* Create skeleton for notes page

* Add eraser and clear buttons for doodle pad

* Add rough saving mechanism

* Create note logic classes

* Add json adapted notes

* Fix up storage issues

* Create new note and open note command, fix app responsiveness

* Fix canvas overflow issue

* Enable editing only on opening note

* Fix tabs to accomodate notes feature

* Add save note command

* Add delete note command and enhance note card

* Update exit command to save note

* Fix bug related to note deletion

* Documentation

alxkohh committed:
Update user guide (#174)

* Add user guide segments for home tab and bookmark commands, help command

* Add screenshot for help tab

* Add description for Program Evaluation Panel

Commit:
Editor now has a line counter and can auto-indent

Editor now has a line counter and can auto-indent

Commit:
Removed unnecessary Topic field

Commit:
Minor changes to UI components

Commit:
Updated CSS stylesheet for line counter component in Editor

Commit:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-F14-1/main

limhawjia committed:
Fix ui

Commit:
Updated ug with new images

limhawjia committed:
Update overview to mention notes

Commit:
updated UG with new pictures

limhawjia committed:
Update ug with notes features

Merge remote-tracking branch 'origin/master' into hj-ug-update

# Conflicts:
#	docs/UserGuide.adoc

Commit:
Updated UG with new images

limhawjia committed:
Change workspace to clear old results when attempting new question

zhangj1an committed:
made loadqns available x flatform. added new data for qn bank

Commit:
Minor changes to UG

zhangj1an committed:
added toy easy qns

zhangj1an committed:
format code

zhangj1an committed:
Merge branch 'master' into zj-loadxplatform

zhangj1an committed:
Merge pull request #186 from AY1920S1-CS2103T-F14-1/zj-loadxplatform

Zj loadxplatform

limhawjia committed:
Merge branch 'master' into hj-ug-update

zhangj1an committed:
Merge pull request #185 from AY1920S1-CS2103T-F14-1/hj-ug-update

Hj ug update

zhangj1an committed:
my change to ug

Commit:
Fixed minor bugs relating to text input control

Commit:
Removed unnecessary functions

Commit:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-F14-1/main

Commit:
Merge branch 'sy-update-editor'

# Conflicts:
#	src/main/resources/view/DukeAcademyStyle.css
#	src/main/resources/view/Editor.fxml
#	src/main/resources/view/MainWindow.fxml

Commit:
Merge branch 'sy-update-ug'

alxkohh committed:
Refactor class name

alxkohh committed:
Edit welcome message

alxkohh committed:
Edit help page descriptions

alxkohh committed:
Checkstyle fix

alxkohh committed:
Merge branch 'master' into ak-refactor-dashboard

alxkohh committed:
Add unit tests for deletebookmark command and its factory

alxkohh committed:
Fix checkstyle

alxkohh committed:
Merge branch 'master' into ak-unit-tests

alxkohh committed:
Fix all tests

limhawjia committed:
Merge pull request #189 from AY1920S1-CS2103T-F14-1/ak-unit-tests

Ak unit tests

alxkohh committed:
Merge branch 'master' into ak-refactor-dashboard

limhawjia committed:
Merge pull request #188 from AY1920S1-CS2103T-F14-1/ak-refactor-dashboard

Ak refactor dashboard

alxkohh committed:
Add UML diagram for Ui

alxkohh committed:
Update Ui class uml diagram

alxkohh committed:
Add explanation for Ui component segment

Commit:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-F14-1/main

# Conflicts:
#	src/main/resources/view/Dashboard.fxml
#	src/main/resources/view/MainWindow.fxml

Commit:
Removed unnecessary files + minor changes

limhawjia committed:
Hj dg update (#190)

* Updated dg up till ProgramSubmissionLogicManger

* Move all dg images to new folder, ammended old descriptions

* Finish dg updates to logic

* Fix outdated Javadoc

* Add instruction to use Main class in ug, improve user feedback

Commit:
Merge branch 'master' into HEAD

# Conflicts:
#	src/main/resources/view/MainWindow.fxml

ning-sy210 committed:
Merge branch 'master' into sy-update-new-editor

Commit:
Added Javadoc for certain methods

Commit:
Merge branch 'sy-update-new-editor' of https://github.com/AY1920S1-CS2103T-F14-1/main into sy-update-new-editor

Commit:
Minor changes to UI

Commit:
Updates to UG new notes features + stage now has updated min size

limhawjia committed:
Added activation bars to sequence diagrams

limhawjia committed:
Change language to be instructive

Commit:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-F14-1/main

Commit:
Merge branch 'sy-update-new-editor'

zhangj1an committed:
Merge pull request #191 from AY1920S1-CS2103T-F14-1/sy-update-new-editor

Sy update new editor

Commit:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-F14-1/main

limhawjia committed:
Finish intro

limhawjia committed:
Imported dg and ug sections

limhawjia committed:
Finish other contributions

limhawjia committed:
Merge remote-tracking branch 'origin/master' into hj-ppp

# Conflicts:
#	docs/UserGuide.adoc

limhawjia committed:
Create outline for contributions

limhawjia committed:
Finish observable section

limhawjia committed:
Finish architecture section

limhawjia committed:
Finish contribution writeup

limhawjia committed:
Finish summary

zhangj1an committed:
temp change

zhangj1an committed:
merged w update

Commit:
Updated documents

limhawjia committed:
Tidied up ug and dg, added images for ppp

Commit:
Documents update

limhawjia committed:
Fixed dimensions of images

alxkohh committed:
Add dg segment for implementation of Bookmark Command

alxkohh committed:
Add tag for ui component

alxkohh committed:
Merge branch 'master' into ak-dg-uml

limhawjia committed:
Cut various parts

alxkohh committed:
Add comments

alxkohh committed:
Merge branch 'ak-dg-uml' of https://github.com/AY1920S1-CS2103T-F14-1/main into ak-dg-uml

zhangj1an committed:
Merge pull request #193 from AY1920S1-CS2103T-F14-1/ak-dg-uml

Ak dg uml

alxkohh committed:
Merge branch 'master' into sy-update-documents

ning-sy210 committed:
Sy update documents (#192)

* Updated documents

alxkohh committed:
Shift photo location

alxkohh committed:
Add ppp

alxkohh committed:
Pull tags from ppp

zhangj1an committed:
updated ug and dg. left with 2 more diagrams

alxkohh committed:
Add pics for ppp

zhangj1an committed:
checkstyle fix

zhangj1an committed:
merged conflicts

Commit:
Included updates + new diagrams

Commit:
Merge branch 'sy-update-documents' of https://github.com/AY1920S1-CS2103T-F14-1/main into sy-update-documents

# Conflicts:
#	docs/DeveloperGuide.adoc

zhangj1an committed:
merged with latest update

Commit:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-F14-1/main

limhawjia committed:
Add alternative implementation

alxkohh committed:
Edit ppp

zhangj1an committed:
reformat code

alxkohh committed:
Fix picture

limhawjia committed:
Merge pull request #194 from AY1920S1-CS2103T-F14-1/zj-updateug

Zj updateug

alxkohh committed:
Reorder ppp includes

alxkohh committed:
Add Ui pic

alxkohh committed:
Fix preview pic

alxkohh committed:
Change preview img

limhawjia committed:
Merge remote-tracking branch 'origin/master' into hj-ppp

# Conflicts:
#	docs/DeveloperGuide.adoc
#	docs/UserGuide.adoc

alxkohh committed:
Change ui img for readme

Commit:
Document updates

Commit:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-F14-1/main

alxkohh committed:
Add manual testing for bookmark and deletebookmark commands

zhangj1an committed:
Merge pull request #195 from AY1920S1-CS2103T-F14-1/hj-ppp

Hj ppp

ning-sy210 committed:
Merge branch 'master' into sy-update-documents

ning-sy210 committed:
Merge branch 'master' into sy-update-documents

alxkohh committed:
Merge branch 'master' into ak-docs

zhangj1an committed:
Merge pull request #196 from AY1920S1-CS2103T-F14-1/sy-update-documents

Sy update documents

alxkohh committed:
Merge branch 'master' into ak-docs

limhawjia committed:
Merge pull request #197 from AY1920S1-CS2103T-F14-1/ak-docs

Ak docs

limhawjia committed:
Remove overlapping parts

zhangj1an committed:
design considerations

zhangj1an committed:
fixed changes

ning-sy210 committed:
Merge pull request #198 from AY1920S1-CS2103T-F14-1/hj-final-ppp

Remove overlapping parts

Commit:
Update existing pictures

zhangj1an committed:
Merge branch 'master' into zj-updateug

limhawjia committed:
Merge pull request #199 from AY1920S1-CS2103T-F14-1/zj-updateug

Zj updateug

ning-sy210 committed:
Merge branch 'master' into sy-updated-pictures

zhangj1an committed:
added missing diagram

alxkohh committed:
ak design

zhangj1an committed:
Merge pull request #200 from AY1920S1-CS2103T-F14-1/sy-updated-pictures

Update existing pictures

ning-sy210 committed:
Merge branch 'master' into zj-updateug

zhangj1an committed:
Merge branch 'master' into zj-updateug

zhangj1an committed:
Merge branch 'zj-updateug' of https://github.com/AY1920S1-CS2103T-F14-1/main into zj-updateug

alxkohh committed:
Merge branch 'master' into ak-doc-update

zhangj1an committed:
Merge pull request #201 from AY1920S1-CS2103T-F14-1/zj-updateug

added missing diagram

limhawjia committed:
Merge pull request #202 from AY1920S1-CS2103T-F14-1/ak-doc-update

ak design

Commit:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-F14-1/main

alxkohh committed:
Fix alignment

alxkohh committed:
Merge branch 'ak-doc-update' of https://github.com/AY1920S1-CS2103T-F14-1/main into ak-doc-update

limhawjia committed:
Merge pull request #203 from AY1920S1-CS2103T-F14-1/ak-doc-update

Ak doc update

alxkohh committed:
gui fix

zhangj1an committed:
Merge pull request #204 from AY1920S1-CS2103T-F14-1/ak-preview

gui preview fix

zhangj1an committed:
updated zj-ppp

zhangj1an committed:
Merge branch 'master' into zj-updateug

zhangj1an committed:
Merge pull request #205 from AY1920S1-CS2103T-F14-1/zj-updateug

Zj updateug

zhangj1an committed:
changed ppp

zhangj1an committed:
Merge branch 'master' into zj-updateug

zhangj1an committed:
Merge pull request #206 from AY1920S1-CS2103T-F14-1/zj-updateug

changed ppp

zhangj1an committed:
Update AboutUs.adoc

zhangj1an committed:
edit names to full name

ning-sy210 committed:
Merge pull request #207 from AY1920S1-CS2103T-F14-1/zhangj1an-patch-1

Update AboutUs.adoc

zhangj1an committed:
file rename

zhangj1an committed:
Merge branch 'master' into zj-updateug

zhangj1an committed:
deleted generated pdf

zhangj1an committed:
Merge pull request #209 from AY1920S1-CS2103T-F14-1/zj-updateug

Zj rename ppp adoc

Commit:
Changed name of files and included picture

ning-sy210 committed:
Merge branch 'master' into sy-fix-broken-github-link

ning-sy210 committed:
Merge pull request #210 from AY1920S1-CS2103T-F14-1/sy-fix-broken-github-link

Sy fix broken GitHub link

Commit:
small changes to AboutUs.adoc

Commit:
Merge branch 'sy-fix-broken-github-link' of https://github.com/AY1920S1-CS2103T-F14-1/main into sy-fix-broken-github-link

# Conflicts:
#	docs/AboutUs.adoc

ning-sy210 committed:
Merge branch 'master' into sy-fix-ppp

ning-sy210 committed:
Merge pull request #211 from AY1920S1-CS2103T-F14-1/sy-fix-ppp

Sy fix ppp

Commit:
Remove empty line

ning-sy210 committed:
Merge pull request #212 from AY1920S1-CS2103T-F14-1/sy-fix-html

Remove empty line
